### PR TITLE
VA-8676 Component Media--image 

### PR DESCRIPTION
**Pull request by amponce**
The media image component is built to accept an image style that allows users to pass a prop to modify the image size.

- 1_1_square_medium_thumbnail:
- 2_1_large 
- 2_1_medium_thumbnail
- 2_3_medium_thumbnail
- 3_2_medium_thumbnail
- 7_2_medium_thumbnail
- full_content_width
- large
- self
- viewport_width

mediaComponent

### CODE REVIEW

File: src/pages/demo/partials/benefitHubsLinks.tsx [hunk 1]
```diff
@@ -19,24 +25,21 @@ const DemoPage = ({ nodes }) => {
...
 export default DemoPage
 
-export async function getStaticProps(context) {
+export async function getStaticProps(
+  context: GetStaticPropsContext
+): Promise<GetStaticPropsResult<BenefitsHubLinksProps>> {
   const params = new DrupalJsonApiParams()
   params
     .addFilter('status', '1')
     .addFilter('field_related_benefit_hubs', null, 'IS NOT NULL')
```
Comment (penny-lischer) [inline src/pages/demo/partials/benefitHubsLinks.tsx:34]: Instead of 'IS NOT NULL' you can use Operators.isNotNull

Comment (amponce) [reply src/pages/demo/partials/benefitHubsLinks.tsx:34]: perfect, nice find we should probably go back and update the some of the other original components with that fix

File: src/components/media/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { screen, render } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import '@testing-library/jest-dom'
+import { MediaImage } from '.'
+
+// Language: typescript
+// Path: src/components/media/index.test.tsx
+
+const media = {
```
Comment (penny-lischer) [inline src/components/media/index.test.tsx:9]: Do we want to name constants uppercase?

File: src/components/media/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    resourceIdObjMeta: {
+      alt: 'pension',
+      title: 'pension',
+      height: '304',
+      width: '456',
+    },
+  },
+}
+
+test('<MediaImage> renders', () => {
```
Comment (penny-lischer) [inline src/components/media/index.test.tsx:39]: Should wrap the tests in a describe

**Pull request merged**